src/Campaigns/Migrations/Tables/CreateCampaignFormsTable.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Give\Campaigns\Migrations\Tables;
+
+use Give\Framework\Database\DB;
+use Give\Framework\Database\Exceptions\DatabaseQueryException;
+use Give\Framework\Migrations\Contracts\Migration;
+use Give\Framework\Migrations\Exceptions\DatabaseMigrationException;
+
+/**
+ * @unreleased
+ * Creates give_core_campaign_forms table
+ */
+class CreateCampaignFormsTable extends Migration
+{
+    /**
+     * @inheritdoc
+     */
+    public static function id(): string
+    {
+        return 'give-campaigns-create-give-core-campaign-forms-table';
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public static function title(): string
+    {
+        return 'Create give_core_campaign_forms table';
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public static function timestamp(): string
+    {
+        return strtotime('2024-08-26 00:00:01');
+    }
+
+    /**
+     * @inheritDoc
+     * @throws DatabaseMigrationException
+     */
+    public function run(): void
+    {
+        global $wpdb;
+
+        $table = $wpdb->prefix . 'give_core_campaign_forms';
```

Comment:
I think we can drop the references to `core` in these migrations (title, tables, etc).

Reply:
I agree!